legacy/messaging/src/main/resources/schema/jboss-as-messaging_1_4.xsd

```diff
@@ -108,6 +108,13 @@
                   </xs:documentation>
               </xs:annotation>
           </xs:element>
+          <xs:element maxOccurs="1" minOccurs="0" name="override-in-vm-security" type="xs:boolean">
```

Comment:
The 1.4 schema must not be changed. We introduced the 1.5 one to port forward the incompatibility introduced in EAP 6.4.